### PR TITLE
[MRG] Add field mocks, beginning of TDD for WLutz

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

**Comment by SimonBiggs** [comment]
@Matthew-Jennings this doesn't actually have any user code in it. In the end this only has two stub functions (that is just their function name and the docstrings for them) as well as a heap of tests for those functions which aren't yet complete.

The real bit of value in this pull request is actually the mock field function. I wanted to be able to quickly mock a field distribution with penumbra and rotation. So that's what the field mocks are for.

I have found them useful in other tasks that are not relevant to WLutz so I would actually be keen for this pull request to merge, and I can pick up where I left off on the WLutz code in a few weeks or so.

But having the mocks in master would be nice :).

**Comment by SimonBiggs** [comment]
Long story short for, keen for this to be reviewed and merged, in full recognition that some of the functions are stubs.

**Comment by SimonBiggs** [comment]
The mocks are what are helpful from this one.

**Comment by SimonBiggs** [comment]
Yes it is unfinished. But the stuff that is actually in the library is just
the mocks if I remember correctly

**Comment by Matthew-Jennings** [comment]
> Yes it is unfinished. But the stuff that is actually in the library is just
the mocks if I remember correctly

Which I'd like to see! (Without writing my own notebook to plot them)

**Comment by SimonBiggs** [comment]
Ahh, sure :) In a taxi at the moment in Sydney on the way to the airport :).

**Comment by Matthew-Jennings** [comment]
> Ahh, sure :) In a taxi at the moment in Sydney on the way to the airport :).

How'd you go?!

**Comment by SimonBiggs** [comment]
Can I call once I get through security?

File: packages/pymedphys_analysis/tests/winstonlutz/test_flip_profile.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        },
+        'top-bot': {
+            'x': [2, 3, 4, 5, 6],
+            'y-top': [5, 6, 7],
+            'y-bot': [-3, -2, -1]
+        }
+    }
+
+
+@pytest.mark.skip(msg="Test not yet finished")
```
[the remark this answers is not in the document]
Comment (SimonBiggs) [reply packages/pymedphys_analysis/tests/winstonlutz/test_flip_profile.py]: Ahh cool, thanks for that :)

**Comment by Matthew-Jennings** [comment]
Call whenever old mate

**Comment by Matthew-Jennings** [comment]
@SimonBiggs, had a pretty decent look now and those mock profiles look gorgeous! Very nice indeed :)

**Comment by SimonBiggs** [comment]
> @SimonBiggs, had a pretty decent look now and those mock profiles look gorgeous! Very nice indeed :)

Thanks Matt, yeah I found them quite helpful for test driven development of the film work actually.

**Comment by SimonBiggs** [comment]
Let me know what else you need on this one for merge. I'm on the road to Brisbane so won't have much (or any) computer access. But should be able to make smaller changes/comments if needed.

**Comment by Matthew-Jennings** [comment]
> Let me know what else you need on this one for merge. I'm on the road to Brisbane so won't have much (or any) computer access. But should be able to make smaller changes/comments if needed.

Probably nothing. Sorry, I just wanted to test the rotation as well :)

**Comment by SimonBiggs** [comment]
> Lovely :)

Thanks Matt :)

**Comment by SimonBiggs** [comment]
Thanks Matt :)

**Pull request merged**